static/js/poll_widget.js

```diff
@@ -172,6 +175,23 @@ export function activate({
         });
     }
 
+    function check_option_button() {
+        const poll_option_input = elem.find("input.poll-option");
+        const option = poll_option_input.val().trim();
+        const options = poll_data.get_widget_data().options;
+
+        if (poll_data.is_option_present(options, option)) {
+            elem.find("button.poll-option").attr("disabled", true);
+            elem.find("button.poll-option").attr(
+                "title",
+                $t({defaultMessage: "A choice that already exists cannot be added."}),
```

Comment:
@alya thoughts on a clearer string here?  Perhaps just "Option already present."

Reply:
Sure, @timabbott 's proposal sounds good to me.

Reply:
@timabbott and @alya I have now changed the tooltip and made a commit.